examples/plot_02_basic_usage.py

```diff
@@ -44,7 +47,7 @@
 # %%
 from skore import load
```

Comment:
I know all this part was there already, but it feels like a duplicate of getting started. Is there a point in having this duplicate doc?

Reply:
It goes more in-depth, maybe it could make sense to merge the 2 examples in 1 for a next PR

Reply:
Can you create a P2 issue for that please? It would also solve the fact that we have to create twice a project, hence with different names.